cura/PrintInformation.py

```diff
@@ -311,18 +311,24 @@ def _updateJobName(self):
         # Only update the job name when it's not user-specified.
         if not self._is_user_specified_job_name:
             if self._pre_sliced:
-                job_name = catalog.i18nc("@label", "Pre-sliced file {0}", base_name)
+                self._job_name = catalog.i18nc("@label", "Pre-sliced file {0}", base_name)
             elif self._application.getInstance().getPreferences().getValue("cura/jobname_prefix"):
                 # Don't add abbreviation if it already has the exact same abbreviation.
                 if base_name.startswith(self._abbr_machine + "_"):
-                    job_name = base_name
+                    self._job_name = base_name
                 else:
-                    job_name = self._abbr_machine + "_" + base_name
+                    self._job_name = self._abbr_machine + "_" + base_name
             else:
-                job_name = base_name
+                self._job_name = base_name
+
+        # In case there are several buildplates, a suffix is attached
+        if self._multi_build_plate_model.maxBuildPlate > 0:
+            connector = "_#"
+            suffix = connector + str(self._active_build_plate + 1)
+            if connector in self._job_name:
+                self._job_name = self._job_name.split(connector)[0] # get the real name
```

Comment:
Isn't this making it "too fancy"?

Reply:
On the other hand, if you use _# already...

Reply:
This if statement will only affect if the builtiplexer feature is enabled.

I did use a connector that is not a common character for naming.